ivadomed/main.py

```diff
@@ -347,6 +348,8 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
     logger.add(str(path_log))
     logger.add(sys.stdout)
 
+    set_seed(context[ConfigKW.SPLIT_DATASET][SplitDatasetKW.RANDOM_SEED])
```

Comment:
Same comments as above with regard to how to best set seeds. Maybe we can think about a higher level key at context level to set the Random Seed for all instead of having to delve down into the SplitDataSet Random Seed level.